repository/reading_history_repository_test.go

```
@@ -0,0 +1,156 @@
+	rh1 := createReadingHistory(t, u.ID, b1.ID)
+	_ = createReadingHistory(t, u.ID, b2.ID)
+
+	now := time.Now().UTC()
+	updateReq := UpdateReadingHistoryRequest{
+		UserID:    u.ID,
+		BookID:    b1.ID,
+		Status:    Reading,
+		StartDate: &now,
+		EndDate:   nil,
+	}
+	_, err := readingHistoryRepo.Update(context.Background(), updateReq)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)